cssGrid/style.css

```diff
@@ -0,0 +1,149 @@
+.container {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(200px , 1fr));
+    /* grid-template-columns: auto; means fill the spaces */
```

Comment:
no need to explain the build-in methods in the comments. Only explain the part of the code you've written that you might think needs some explanation